build.gradle

```diff
@@ -101,11 +102,11 @@ dependencies {
     testImplementation group: "io.kestra", name: "repository-memory", version: kestraVersion
 
     // testcontainers
-    testImplementation "org.testcontainers:testcontainers:1.18.1"
-    testImplementation "org.testcontainers:junit-jupiter:1.18.1"
-    testImplementation "org.testcontainers:localstack:1.18.1"
```

Comment:
why downgrading ?

Reply:
1.18.1 => 1.18.3 seems like an upgrade to me